src/main/java/pl/nogacz/checkers/board/Board.java

```diff
@@ -404,4 +425,109 @@ public static boolean isThisSameColor(Coordinates coordinates, PawnColor color)
     public static PawnClass getPawn(Coordinates coordinates) {
         return board.get(coordinates);
     }
+
+     class Menu extends JFrame implements ActionListener{
+        public static final int WIDTH = 500;
+        public static final int HEIGHT = 500;
+        private JButton Resume_button = new JButton("Resume");
+        private JButton NewGame_button = new JButton("New Game");
+        private JButton Exit_button = new JButton("EXIT");
+    
+        public Menu(Board board) {
+            //Initialize
+            super("Game Menu");
+            setSize(WIDTH,HEIGHT);
+            setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE); // dont need that but maybe someone want to remove undecorated
+            this.setUndecorated(true);
+            this.setBackground(new Color(0, 0, 0,100));
+            this.add(Resume_button);
+            this.add(NewGame_button);
+            this.add(Exit_button);
+            //Set Bounds
+            String exit_path = this.getClass().getClassLoader().getResource("button_exit.png").getFile();
+            String resume_path = this.getClass().getClassLoader().getResource("button_resume.png").getFile();

```

Comment:
Burada tekrar eden kodlar var. Bu kodu bir metot içerisine al ve her bir değişken için o metodu çağır. Kodu sadeleşir okuması anlaması kolaylaşır.

getImageIcon(String name) {
  File f = this.getClass().getClassLoader().getResource(name).getFile();
  return new ImageIcon(f);
}
}

exit_image_file

Reply:
Done